base/pgsql/pgsql.Deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       initContainers:
       - name: correct-data-dir-permissions
         image: sourcegraph/alpine:3.12@sha256:ce099fbcd3cf70b338fc4cb2a4e1fa9ae847de21afdb0a849a393b87d94fb174
-        command: ["sh", "-c", "if [ -d /data/pgdata-12 ]; then chmod 750 /data/pgdata-12; fi"]
```

Comment:
We should also do this for the https://github.com/sourcegraph/deploy-sourcegraph/pull/2902/files#diff-b822422461c5c4358eb788df612e854aa5d5701144feebb0b863c0655fe6a2f5R30

Reply:
> This needs to be backported if we decide to do this, correct?
> Fixes [sourcegraph/sourcegraph#20425](https://github.com/sourcegraph/sourcegraph/issues/20425)

This PR is to be merged only against 3.27...which is essentially the backport. It's the only release affected by this. The releases either side of this don't have any migration tasks. 

In my opinion, we should change this for the other db's as you correctly pointed out ( will do this shortly ) then tag the newer commit with 3.27.5. This is an infra level change that I don't think actually warrants a new patch release as there are no application level code changes. 

I recall this being discussed once before, I haven't searched for any official docs yet but will aim to do so this afternoon in order to clarify

Reply:
I don't think we need to tag a new release as all of the images are the same. We would only need to move the tag to the new commit. All of the current documentation references that tag, so there's no need update anything else.